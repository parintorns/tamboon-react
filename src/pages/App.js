import React from 'react'

function App() {
  return <div>Omise Tamboon React</div>
}

export default App
