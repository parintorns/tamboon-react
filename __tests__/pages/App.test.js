import React from 'react'
import { shallow } from 'enzyme'

import App from '../../src/pages/App'

let wrapper

const props = {}

beforeEach(() => {
  wrapper = shallow(<App {...props} />)
})

describe('Render', () => {
  it('should render App', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
