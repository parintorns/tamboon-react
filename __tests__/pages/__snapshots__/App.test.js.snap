// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render App 1`] = `
"<div>
  Omise Tamboon React
</div>"
`;
